k-distribution/src/main/scripts/lib/pyk/kastManip.py

```diff
@@ -508,9 +590,377 @@ def removeSourceMap(k):
     def _removeSourceMap(att):
         if isKAtt(att):
             atts = att['att']
-            newAtts = { }
+            newAtts = {}
             for attKey in atts.keys():
                 if attKey != 'org.kframework.attributes.Source' and attKey != 'org.kframework.attributes.Location':
                     newAtts[attKey] = atts[attKey]
-            return KAtt(atts = newAtts)
+            return KAtt(atts=newAtts)
     return onAttributes(k, _removeSourceMap)
+
+
+def boolToken(b):
+    return KToken(str(b).lower(), 'Bool')
+
+
+def intToken(i):
+    return KToken(str(i), 'Int')
+
+
+def stringToken(s):
+    return KToken('"' + s + '"', 'String')
+
+
+def ltInt(i1, i2):
+    return KApply('_<Int_', [i1, i2])
+
+
+def leInt(i1, i2):
+    return KApply('_<=Int_', [i1, i2])
+
+
+def mlEquals(a1, a2):
+    return KApply('#Equals', [a1, a2])
+
+
+def mlEqualsTrue(b):
+    return mlEquals(boolToken(True), b)
+
+
+def mlTop():
+    return KConstant('#Top')
+
+
+def mlBottom():
+    return KConstant('#Bottom')
+
```

Comment:
Do you think these belong here or in something like `k-prelude.py`?

Reply:
Can happen in later PR: https://github.com/runtimeverification/k/issues/2444

Reply:
Done: https://github.com/runtimeverification/k/pull/2443/commits/1fba293d9f1720b79b742f86b4df290d60cff931